src/Logging/Microsoft.Omex.Extensions.Logging.csproj

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <TargetFrameworks>$(LibraryTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup Condition="$(TargetFramework.StartsWith(`netstandard`))">
```

Comment:
MSBuild defines the default constant `NETSTANDARD` no need to create another one https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-a-target-framework

Reply:
Thanks Andrey I was looking for something like that but didn't come across that doc. :)